client_it_test.go

```diff
@@ -102,6 +262,16 @@ func TestClientRunning(t *testing.T) {
 	})
 }
 
+func TestClient_Name(t *testing.T) {
+	clientName := "test-client-name"
+	cnfCallBack := func(config *hz.Config) {
```

Comment:
How about just `cb` instead of `cnfCallBack`?

Reply:
Fixed: [88b6f60](https://github.com/hazelcast/hazelcast-go-client/pull/809/commits/88b6f60a915c3d0237ae5f1fb97bdbda52cedfbd)